django/db/backends/oracle/operations.py

```diff
@@ -310,17 +312,88 @@ def savepoint_create_sql(self, sid):
     def savepoint_rollback_sql(self, sid):
         return convert_unicode("ROLLBACK TO SAVEPOINT " + self.quote_name(sid))
 
+    @cached_property
```

Comment:
Tables created in migrations are always dropped in order for `--keepdb` to work in next runs. I'll open a new PR and give a Oracle a test run. I managed to get [a successful build](http://djangoci.com/view/PR%20Builders/job/pull-requests-oracle/103/console) before closing this PR.